repository/hashstack/metadata.json

```diff
@@ -1,19 +1,132 @@
 {
   "id": "hashstack",
   "displayName": "Hashstack",
-  "host": "zk.hashstack.finance",
-  "description": "A non-custodial Zk-native money market protocol enabling secure under-collateralized loans.",
+  "host": "hstk.fi/hashstack",
+  "description": "Hashstack users get 5x their collateral as debt for yield farming & trading capital.",
   "categories": [
-    "defi"
+    "lending",
+		"defi"
   ],
   "contracts": [
     {
-      "tag": "hashstack",
+      "tag": "hashstack_rBTC",
       "addresses": {
-        "mainnet-alpha": [
-          "0x03dcf5c72ba60eb7b2fe151032769d49dd3df6b04fa3141dffd6e2aa162b7a6e"
+        "mainnet": [
```

Comment:
pls change all `"mainnet"` to `"mainnet-alpha"` since "mainnet" is not a valid starknet chain name